docs/installation/setup-aws.rst

```diff
@@ -47,7 +49,7 @@ You can also refer to `the full documentation for interacting with EC2 from Amaz
    * **Configure security group**:
       
      * If you wish to customize the instance's security settings, make sure to permit SSH access from the administrators machine (for example, your laptop).
-     * To enable Flocker agents to communicate with the :ref:`Flocker control service <enabling-control-service>` and for external access to the API, add a custom TCP security rule enabling access to ports 4523-4524.
+     * To enable Flocker agents to communicate with the Flocker control service and for external access to the API, add a custom TCP security rule enabling access to ports 4523-4524.
      * Keep in mind that (quite reasonably) the default security settings firewall off all ports other than SSH.
```

Comment:
This sentence is a little confusing. Can we please consider rewording it to ``Keep in mind that, quite reasonably, the default security setting comes with firewall disabled on all ports other than SSH port``? Thanks!